# Worksheet06

This is for your joy, amusement, and edification. Don't think of it as an assignment. Think of it as a chance to exercise your skills. 

You may collaborate with friends, refer to notes, search the web, or use any other resources as you work on the problems below.

Relevant notes are from September 18th.

1) Create a function that returns the remainder of two numbers

In [1]:
# write your code here
def remainder(x1, x2):
    r = x1%x2 #replace this line with the remainder of x1/x2
    return r

2) Use your function to compute the remainder of 11 divided by 3

In [2]:
remainder(11,3)# write your code here

2

I got 2

3) Define a function called "half" that returns half the value of a number x

In [3]:
def half(x):
    r=x//2
    return r# write your code here
# should start with def half(x):

4) Use your function to compute half of 100

In [4]:
half(100)# write your code here

50

I got 50

5) Write a function called "inchestocm" that converts inches to centimeters (1 inch = 2.54 centimeters)

In [5]:
def inchestocm(x):
    r=x*2.54
    return r# write your code here
# should start with def inchestocm(xinches):

6) Use your function to convert 10 inches to centimeters

In [6]:
inchestocm(10)# write your code here

25.4

I got  25.4 centimeters

7) Write a function called "feettoinches" that converts feet to inches

In [7]:
def feettoinches(x):
    r= x*12
    return r# write your code here

8) Using feettoinches and inchestocm, create a function called "feettocm" that converts feet to centimeters

In [8]:
# write your code here
def feettocm(xfeet):
    xinches = xfeet*12 #replace this. Use feettoinches to convert xfeet to inches
    xcm = xinches*2.54 #replace this. Use inchestocm to convert xinches to centimeters
    return xcm

9) Use feettocm to convert 4 feet to centimeters

In [9]:
feettocm(4)# write your code here

121.92

I got 121.92

When you run the cell below you should get a **Submit this notebook** button. Click that button, login, and your notebook should be recorded.

In [10]:
# do not change the code below
import comp116

comp116.submit('ws6')